README.md

```diff
@@ -135,14 +135,14 @@ before installing
 ## NewsFeed
 
 Back In Time has a RSS feed
-http://feeds.launchpad.net/backintime/announcements.atom
+https://feeds.launchpad.net/backintime/announcements.atom
 
 ## Contribute
 
-There is a dev-docu on http://backintime-dev.rtfd.org/
+There is a dev-docu on https://backintime-dev.rtfd.org/
```

Comment:
That link does not work because the cert is only valid for `*.readthedocs.org`. Please change it to `https://backintime-dev.readthedocs.org` so I can merge this